### ISSUE TITLE
Message Send Timer (idk how to word this)

### ISSUE THREAD

**Issue by Saberconn** [Aliucord/plugin-requests#465]
### Discord Account

Wess in a Bass Clef#2682

### Plugin Description

Sometimes you type in a message and as soon as you click enter, you instantly regret it. Sure, some people have the benefit of slow wifi, but not everyone is cursed with slow wifi. If we have a plugin that sets a timer before it sends, that would make lots of people not hate each other because of a stupid message we regretfully sent.

Thank you for listening to my ted talk

### Go into more detail...

Maybe add a setting for how long you want for a message to send

### Request Agreement

- [X] I did indeed check to make sure my plugin request is original!

**Comment by Vendicated** [comment]
so basically it would delay sending for a few seconds and then you can press to cancel it?

**Comment by Saberconn** [comment]
Yes, it would delay the process of sending the message with an option to cancel. Yes you can delete a message but sometimes people will see it before you can delete it
